subsys/bluetooth/controller/hci_internal.h

```diff
@@ -24,6 +27,32 @@
  */
 int hci_internal_cmd_put(uint8_t *cmd_in);
 
+/** A user implementable HCI command handler
```

Comment:
Maybe add a short description here what that handler is supposed to do. I.e. that it should generate an event.

Reply:
It should only handle the command. It does not need to generate an event. That is done automatically.
I can add a few more details